pkg/k8shandler/cluster.go

```diff
@@ -207,16 +209,64 @@ func getNodeUpgradeInProgress(cluster *v1alpha1.Elasticsearch) NodeTypeInterface
 	return nil
 }
 
+func setUUIDs(cluster *v1alpha1.Elasticsearch) {
+
+	for index := 0; index < len(cluster.Spec.Nodes); index++ {
+		if cluster.Spec.Nodes[index].GenUUID == nil {
+			uuid, err := utils.RandStringBytes(8)
```

Comment:
I'm not sure if its better or not, but thoughts about hashing maybe the name and/or namespace as the identifier instead of random?  Presumably they are already unique and the repeatible/consistent nature may be more advantageous then random